cdap-proto/src/main/java/io/cdap/cdap/proto/artifact/AppRequest.java

```diff
@@ -54,12 +56,15 @@ public AppRequest(ArtifactSummary artifact, @Nullable T config, @Nullable String
   }
 
   public AppRequest(ArtifactSummary artifact, @Nullable T config, @Nullable PreviewConfig preview,
-                    @Nullable String ownerPrincipal, @Nullable Boolean updateSchedules) {
+    @Nullable String ownerPrincipal, @Nullable Boolean updateSchedules) {
```

Comment:
Don't change the indentation. The old one is correct.

Reply:
Fixed